docs/README.md

```diff
@@ -1,47 +1,190 @@
+
+   pod.yaml: (object: <no namespace>/security-context-demo /v1, Kind=Pod) container    "sec-ctx-demo" has cpu limit 0 (check: unset-cpu-requirements, remediation: Set    your container's CPU requests and limits depending on its requirements. See    https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/   #requests-and-limits for more details.)
+   
+   pod.yaml: (object: <no namespace>/security-context-demo /v1, Kind=Pod) container    "sec-ctx-demo" has memory limit 0 (check: unset-memory-requirements, remediation:    Set your container's memory requests and limits depending on its requirements.    See https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/   #requests-and-limits for more details.)
+   
+   Error: found 3 lint errors
+   ```
+
+
+#### ** Helm **
+
+To run KubeLinter on Helm charts, provide a path to the directory which contains
+the `Chart.yaml` file. For example, consider running KubeLinter on a sample Helm
+chart:
+
+1. Create a new Helm chart:
+   ```bash
+   helm create helm-chart-sample
+   ```
+1. To lint this Helm chart with KubeLinter, run the following command:
+   ```bash
+   kube-linter lint helm-chart-sample/
+   ```
+1. KubeLinter runs the default checks and reports errors.
+   ```
+   helm-chart-sample/helm-chart-sample/templates/tests/test-connection.yaml: (object: <no namespace>/test-release-helm-chart-sample-test-connection /v1, Kind=Pod) container "wget" does not have a read-only root file system (check: no-read-only-root-fs, remediation: Set readOnlyRootFilesystem to true in your container's securityContext.)
+
+   helm-chart-sample/helm-chart-sample/templates/tests/test-connection.yaml: (object: <no namespace>/test-release-helm-chart-sample-test-connection /v1, Kind=Pod) container "wget" is not set to runAsNonRoot (check: run-as-non-root, remediation: Set runAsUser to a non-zero number, and runAsNonRoot to true, in your pod or container securityContext. See https://kubernetes.io/docs/tasks/configure-pod-container/security-context/ for more details.)
+
+   helm-chart-sample/helm-chart-sample/templates/tests/test-connection.yaml: (object: <no namespace>/test-release-helm-chart-sample-test-connection /v1, Kind=Pod) container "wget" has cpu request 0 (check: unset-cpu-requirements, remediation: Set your container's CPU requests and limits depending on its requirements. See https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits for more details.)
+   
+   ...
+
+   Error: found 12 lint errors
+   ```
+
+
+<!-- tabs:end -->
+
+
+For more details about using and configuring KubeLinter, see the
+[Using KubeLinter](/using-kubelinter) topic.
+
+# Community
+
+To engage with the KubeLinter community, including maintainers and other
```

Comment:
Can we add a Slack logo or emoji on this line? (I remember seeing it on one of your earlier previews)